InterfacePython/TasmanianSG.py

```diff
@@ -2353,6 +2411,15 @@ def makeWaveletGrid(iDimension, iOutputs, iDepth, iOrder=1, liLevelLimits=[]):
     grid.makeWaveletGrid(iDimension, iOutputs, iDepth, iOrder, liLevelLimits)
     return grid
 
+def makeGridFromCustomTabulated(iDimension, iOutputs, iDepth, sType, iCustomTabulated, liAnisotropicWeights=[], liLevelLimits=[]):
```

Comment:
The naming convention is a bit old, but the first character in the name indicates the type:
```
iOutputs is an integer
sType is a string
pGrid is an object of some other type (e.g., sparse grid)
liLevelLimits is a list of integers, or a numpy array
llfX is a double-list or more specifically 2D-array of floating point numbers
pCustomRule is an object of type Tasmanian.CustomTabulated()
```